java/code/src/com/suse/manager/matcher/MatcherJsonIO.java

```diff
@@ -350,7 +351,7 @@ public String generateMatcherInput(boolean includeSelf, String arch, boolean sel
 
         // add SUSE Manager entitlements
         return concat(
-                products.stream().map(SUSEProduct::getProductId),
+                server.isPayg() ? Stream.empty() : products.stream().map(SUSEProduct::getProductId),
```

Comment:
@hustodemon this means that those systems will get no product rather than free products from the matcher's POV. Are you explicitly OK with that?

Reply:
We had a discussion the other day together with @mcalmer and @hustodemon, and the conclusion was: for PAYG instances we only want to report the "SUMA Lifecycle" subscription and also the "Monitoring" one if that's the case. We're excluding all base installed products.

Reply:
Exactly, only the manager-related subscriptions (lifecycle + monitoring). That should be the goal.